integration/test/performancetest/performance_test.go

```diff
@@ -1,59 +1,158 @@
-// go:build linux && integration
+//go:build linux && integration
 // +build linux,integration
 
 package performancetest
 
-import(
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"sync"
 	"testing"
 	"time"
-	"log"
-	"context"
+
 	"github.com/aws/amazon-cloudwatch-agent/integration/test"
 )
 
 const (
-	configPath = "resources/config.json"
 	configOutputPath = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
-	agentRuntimeMinutes = 5 //20min desired but 5mins for testing purposes 
+	agentRuntimeMinutes = 5 //20 mins desired but 5 mins for testing purposes
 	DynamoDBDataBase = "CWAPerformanceMetrics"
-
 )
 
 func TestPerformance(t *testing.T) {
+	configFiles := []string {
+		"./resources/config10Logs.json",
+		"./resources/config100Logs.json",
+		"./resources/config1000Logs.json"
+	}
+
+	tpsVals := []int {
+		10,
+		100,
+		1000
+	}
+	
```

Comment:
Would be better to add a `defer deleteLogGroup(instanceID)` (another function since your test case will push a lots of logs into different log stream)?